srcpkgs/ly/files/ly/conf

```diff
@@ -0,0 +1,14 @@
+if [ -x /sbin/agetty -o -x /bin/agetty ]; then
+	# util-linux specific settings
+	if [ "${tty}" = "tty1" ]; then
+		GETTY_ARGS="--noclear"
+	fi
+fi
+
+BAUD_RATE=38400
+TERM_NAME=linux
+
+StandardInput=tty
+TTYPath=/dev/tty7
+TTYReset=yes
+TTYVHangup=yes 
```

Comment:
Are these being used anywhere?

Reply:
BAUD_RATE and TERM_NAME are used in the `run` file. The others are inherited from the systemD service file. They don't appear to be used anywhere, but they do correspond to commands that should probably be added to the `run` file. In this case, the commands would be changing the virtual terminal to whichever one is specified there and issuing `reset` to clear the screen. At least, that's what I got from [this article](https://alan-mushi.github.io/2014/10/26/execute-an-interactive-script-at-boot-with-systemd.html).

Reply:
Ok, so perhaps add a `reset` command before launching ly. Not sure what `hangup` gives us.